### DISCUSSION TITLE
Api filter regarding a specific record in another collection

### DISCUSSION THREAD

**Discussion by hugoschreib** [pocketbase/pocketbase#2296] [Q&A] [answered]
I have one auth collection that stores my users. Each user has a field "role" that is either "normal" or "super". I also have one base collection named "public" that is read only for "normal" users and read-write for "super" users. In the public collection there is a specific record that aims to regulate new user, it contains a string that is by default "open" => "I accept new users" and that can be modified to "close" => "I don't want the creation of any new users" by "super" users. Is there any way that in the auth collection i specify into the create api filter a rule that reads from the public collection to allow or not the creation of new users ?

**Comment by ganigeorgiev** [comment]
The description is a little confusing and I'm not sure that I understand the use case, but if you want to apply a condition on other non-directly related collection field, you can try with the `@collection.*` filter. Something like:

```js
@collection.public.someField ?= "open"
```

_(aka. at least one of "public.someField" records value is equal to "open")_

**Reply by hugoschreib** [reply]
Firslty, I apologize for the quite confusing question. Then I would like you to know that your answer helped a lot. However it didn"t answer all of my question so I will try to reformulate : I don't want to look for at least one of "public.someField" records value is equal to "open" but rather public.thisRecord.someField" records value is equal to "open" and "thisRecord" is a known id of the collection

**Reply by ganigeorgiev** [reply, marked as answer]
>  I don't want to look for at least one of "public.someField" records value is equal to "open" but rather public.thisRecord.someField" records value is equal to "open" and "thisRecord" is a known id of the collection

You can combine multiple `@collection.` conditions, aka.:

```js
@collection.public.id ?= "someRecordId" &&
@collection.public.someField ?= "open"
```

_Translates to: the create rule will be true if at least one `public` record has id matching "someRecordId" AND the `someField` of that record is also equal to "open"._

**Reply by hugoschreib** [reply]
That is perfectly clear, thanks !
